python/ray/tune/tune.py

```diff
@@ -45,10 +107,22 @@ def run_experiments(experiments, scheduler=None, with_server=False,
     runner = TrialRunner(
         scheduler, launch_web_server=with_server, server_port=server_port)
 
-    for name, spec in experiments.items():
-        for trial in generate_trials(spec, name):
+    if type(experiments) is dict:
+        for name, spec in experiments.items():
+            for trial in generate_trials(spec, name):
+                trial.set_verbose(verbose)
```

Comment:
nit: you're doing the same `.set_verbose()` and `add_trial()` in each of these loop bodies. not a problem for now, but maybe consider pulling out a helper func if things get more complicated in the future.

Reply:
Good point; I'll leave it as for now but will most likely do that in the future